### ISSUE TITLE
Certains mots-clés non pris en compte

### ISSUE THREAD

**Issue by jdesboeufs** [etalab/geo.data.gouv.fr#269]
Dans le catalogue Adour-Garonne, certains mots-clés ne sont pas pris en compte correctement.

Exemple : http://catalogue.adour-garonne.eaufrance.fr/catalog/srv/fre/catalog.search#/metadata/46aa03c0-09a3-11de-97dd-001517506978

<img width="618" alt="capture d ecran 2017-02-03 a 12 07 31" src="https://cloud.githubusercontent.com/assets/1231232/22589222/7f2108ee-ea09-11e6-9c98-7b5ba2943f0b.png">

Résultat : https://inspire.data.gouv.fr/datasets/cc8582df4839258bc5162fd2f4b5d897773e2aae

<img width="135" alt="capture d ecran 2017-02-03 a 12 07 43" src="https://cloud.githubusercontent.com/assets/1231232/22589228/8603ba6c-ea09-11e6-978f-7d266e77f585.png">

Il manque plusieurs mots-clés, dont le mot-clé "données ouvertes".

**Comment by xcaeag** [comment]
Sans comprendre pourquoi, la fiche citée en exemple contient désormais les mots-clés. Les mots séparés par des virgules ont été isolés, la fiche source a donc subi des modifications entre temps.

Mais d'autres fiches sont encore concernées par le pb, ex :
http://catalogue.adour-garonne.eaufrance.fr/catalog/srv/fre/catalog.search?node=srv#/metadata/b8303da0-03b7-11dc-bdfd-001676d071d9

Merci pour ce travail.

**Comment by jdesboeufs** [comment]
Avez-vous fait des manipulations sur cette dernière fiche cet après-midi ?
Il y a encore 1 heure je voyais bien le problème, maintenant il n'apparait plus non plus !!

**Comment by xcaeag** [comment]
Oui, je fais une passe assez exhaustive sur nos fiches pour homogénéiser différents compartiments… sur celle-ci, je ne vois pas ce qui a bougé. 
Il me semble que le simple fait de sauver une fiche sous geonetwork en corrige des anomalies.

En voici une autre :

https://inspire.data.gouv.fr/datasets/383c8b67b2bdcb1f7d230261a8e679836f87ac22
http://catalogue.adour-garonne.eaufrance.fr/catalog/srv/fre/catalog.search#/metadata/b2d17406-57eb-4b62-8655-54f1e181aa28

**Comment by jdesboeufs** [comment]
Je pense avoir trouvé d'où ça vient.
Ces fiches sont présentes sur un catalogue qui se comporte anormalement [PICTO Occitanie](https://inspire.data.gouv.fr/catalogs/5891a82131ba67bce9289b16), et qui ne supporte plus certains paramètres pourtant conformes INSPIRE.
Votre fiche remonte donc de manière incomplète de ce catalogue, et comme les dates de mise à jour coïncident, c'est par malchance la fiche de PICTO qui est prise en compte.

Je continue d'investiguer.
